src/status_im/ui/screens/add_new/new_chat/styles.cljs

```diff
@@ -0,0 +1,9 @@
+(ns status-im.ui.screens.add-new.new-chat.styles
+  (:require [status-im.ui.components.colors :as colors]))
+
+(def error-message
+  {:margin-horizontal 14
```

Comment:
Do we have specific error message handling?

Reply:
yes

Reply:
this is specific error message used only here